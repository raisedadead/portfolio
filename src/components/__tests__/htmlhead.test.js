import React from 'react'
import renderer from 'react-test-renderer'
import HtmlHead  from '../htmlhead'
const metaData = require(`../../../gatsby-config`).siteMetadata

describe(`HtmlHead`, () =>
  it(`renders correctly`, () => {
    const data = {
      site: {
        siteMetadata: {
          defaultTitle: metaData.defaultTitle,
          description: metaData.description,
          lang: metaData.lang,
          locale: metaData.locale,
          icon: metaData.icon,
          siteUrl: metaData.siteUrl,
          titleTemplate: metaData.titleTemplate,
          twitterCreator: metaData.twitterCreator  
        }
      }
    }
    const tree = renderer.create(<HtmlHead data={data} />).toJSON()
      expect(tree).toMatchSnapshot()
  })
)
