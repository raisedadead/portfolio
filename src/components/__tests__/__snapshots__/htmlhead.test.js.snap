// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HtmlHead renders correctly 1`] = `null`;
